web/docs/database/backends.md

```diff
@@ -1,3 +1,329 @@
 ---
-title: Database Backends
+title: Databases
 ---
+
+[Entities](/docs/database/entities.md), [Operations](/docs/database/operations.md) and [Automatic CRUD](/docs/database/crud.md) together make a high-level interface for working with your app's data. Still, all that data has to live somewhere, so let's see how Wasp deals with databases.
+
+## Supported Database Backends
+
+Wasp supports multiple database backends. We'll list and explain each one.
+
+### SQLite
+
+The default database Wasp uses is [`SQLite`](https://www.sqlite.org/index.html).
+
+SQLite is a great way for getting started with a new project because it doesn't require any configuration, but Wasp can only use it in development. Once you want to deploy your Wasp app to production, you'll need to switch to `PostgreSQL` and stick with it.
+
+Fortunately, migrating from SQLite to PostgreSQL is pretty simple, and we have [a guide](#migrating-from-sqlite-to-postgresql) to help you.
+
+### PostgreSQL
+
+[PostgreSQL](https://www.postgresql.org/) is a real production database.
+
+To use Wasp with PostgreSQL, you'll have to ensure a database instance is running during development. Wasp needs access to your database for commands such as `wasp start` or `wasp db ...` and expects to find a connection string in the `DATABASE_URL` environment variable.
+
+We cover all supported ways of connecting to a database in [the next section](#connecting-to-a-database).
+
+### Migrating from SQLite to PostgreSQL
+
+To run your Wasp app in production, you'll need to switch from `SQLite` to `PostgreSQL`.
+
+1. Set `app.db.system` to `PostgreSQL`.
+2. Delete all old migrations, since they are SQLite migrations and can't be used with PostgreSQL: `rm -r migrations/`.
```

Comment:
```suggestion
2. Delete all old migrations, since they are SQLite migrations and can't be used with PostgreSQL:

\`\`\`
rm -r migrations/`
\`\`\`
```